addons/html_editor/static/tests/odoo_collaboration.test.js

```diff
@@ -1171,7 +1172,16 @@ describe("Selection", () => {
             peers.p2.plugins.collaborationSelection.selectionInfos.get("p1").selection.anchorOffset
         ).toBe(1);
         peers.p1.plugins.delete.delete("backward", "character");
-        advanceTime(100);
+        await waitUntil(() => {
+            let answer =
+                peers.p2.plugins.collaborationSelectionAvatar.selectionInfos.get("p1").selection
+                    .anchorOffset == 0;
+            answer =
+                answer &&
+                peers.p2.plugins.collaborationSelection.selectionInfos.get("p1").selection
+                    .anchorOffset == 0;
+            return answer;
+        });
```

Comment:
The four `answer` with their long (re)definitions makes it hard to read. I suggest using the following form instead:
```suggestion
await waitUntil(() => {
    const relevantName =
        peers.p2.plugins.collaborationSelectionAvatar.selectionInfos.get("p1").selection.anchorOffset == 0;
    const otherRelevantName =
        peers.p2.plugins.collaborationSelection.selectionInfos.get("p1").selection.anchorOffset == 0;
    return relevantName && otherRelevantName;
});
```